src/astra_driver.cpp

```diff
@@ -417,8 +445,10 @@ void AstraDriver::imageConnectCb()
     {
       device_->setColorFrameCallback(boost::bind(&AstraDriver::newColorFrameCallback, this, _1));
 
-      ROS_INFO("Starting color stream.");
-      device_->startColorStream();
+      if (enable_streaming_) {
```

Comment:
Is it better to use different `enable_streaming_` parameter for the Color, IR and Depth streaming?